site/src/pages/CreateTemplatePage/CreateTemplateForm.tsx

```diff
@@ -224,6 +224,18 @@ export const CreateTemplateForm: FC<CreateTemplateFormProps> = ({
   const { t } = useTranslation("createTemplatePage")
   const { t: commonT } = useTranslation("common")
 
+  useEffect(() => {
+    if (error) {
+      window.scrollTo(0, 0)
+    }
+  }, [error])
+
+  useEffect(() => {
+    if (jobError) {
+      window.scrollTo(0, document.body.scrollHeight)
+    }
+  }, [logs])
```

Comment:
This should fix lint
```suggestion
  }, [logs, jobError])
```